pkg/generated/clientset/versioned/typed/serviceca/v1/fake/fake_serviceca.go

```diff
@@ -123,7 +123,7 @@ func (c *FakeServiceCAs) DeleteCollection(options *v1.DeleteOptions, listOptions
 // Patch applies the patch and returns the patched serviceCA.
 func (c *FakeServiceCAs) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (result *servicecav1.ServiceCA, err error) {
 	obj, err := c.Fake.
-		Invokes(testing.NewRootPatchSubresourceAction(servicecasResource, name, pt, data, subresources...), &servicecav1.ServiceCA{})
+		Invokes(testing.NewRootPatchSubresourceAction(servicecasResource, name, data, subresources...), &servicecav1.ServiceCA{})
```

Comment:
Please put generated changes into a separate commit.

Reply:
now in 03446ad1